mlibrary/get-this
# Login update

# Overview
* Implements the logic needed to enable the latest omniauth. 
* Did some refactoring was done so that I didn't need to perpetuate code smells.
* Updated ruby packages

## Anything else?
@erinesullivan  Can you verify that this works and that the `erb` and `js` are OK?

## Testing
List instructions on how to test the pull request. Some examples:

- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`)
  - Break the new/updated unit tests to make sure they're working properly.
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- _How to use the feature_.
